geo-data-server/src/main/java/org/gridsuite/geodata/server/GeoDataService.java

```diff
@@ -52,6 +58,18 @@ private Set<String> toCountryIds(Collection<Country> countries) {
         return countries.stream().map(Country::name).collect(Collectors.toSet());
     }
 
+    @PostConstruct
+    public void init() {
+        defaultSubstationsGeoData = new DefaultSubstationGeoDataByCountry();
+        try {
+            ClassLoader classLoader = getClass().getClassLoader();
+            File file = new File(classLoader.getResource("config/substationGeoDataByCountry.json").getFile());
+            defaultSubstationsGeoData = mapper.readValue(file, DefaultSubstationGeoDataByCountry.class);
```

Comment:
This is not the correct way to load resources: get an input stream like getClasss().getResourceAsStream
and then use the readValue of object mapper taking an inputstream

Reply:
Done